lib/amazon-pricing/rds-price-list.rb

```diff
@@ -20,9 +20,12 @@ def initialize
       :mysql => {:mysql=>["standard","multiAZ"]},
       :postgresql => {:postgresql=>["standard","multiAZ"]},
       :oracle => {
-        :oracle_se1=>["li-standard","li-multiAZ","byol-standard","byol-multiAZ"],
+        # AWS: Oracle - Enterprise Edition, Standard Edition Two, Standard Edition, Standard Edition One.
+        # License Included Charges supported: Standard Edition One, Standard Edition Two
+        # BYOL Charges supported: do not vary by edition for BYOL Amazon RDS pricing.
         :oracle_se=>["byol-standard","byol-multiAZ"],
-        :oracle_ee=>["byol-standard","byol-multiAZ"]
+        :oracle_se1=>["li-standard","li-multiAZ"],
+        :oracle_se2=>["li-standard","li-multiAZ"]
```

Comment:
This is intentional:  see @@RESERVED_DB_WITH_SAME_PRICING2 as :oracle_se and :oracle_ee share the same pricings.